modules/bloxone_infra_host_azure/.terraform-docs.yml

```diff
@@ -0,0 +1,12 @@
+formatter: "markdown table"
+
+header-from: main.tf
+
+sections:
+  hide:
+    - modules
+
+output:
+  file: "README.md"
+  mode: inject
+
```

Comment:
@mathewab Can this file be moved under the modules directory now instead of being in both the subdirectories?

Reply:
No, the tool doesn't work if the file is moved there. It has to be under each of the module (except if it is a child module). All our modules are root modules as of now.